cadc-access-control-server/src/main/java/ca/nrc/cadc/ac/server/oidc/AuthorizeAction.java

```diff
@@ -218,6 +220,9 @@ private void doOpenIDCodeFlow() throws Exception {
                 redirect.append("&state=");
                 redirect.append(state);
             }
+            redirect.append("&client=").append(rp.getClientDescription());
```

Comment:
Not sure about `state` above, but these `client` and `claims` param values need to be URL encoded with `NetUtil.encode(..)`